index.html

```diff
@@ -1477,9 +1482,15 @@ <h2>
         </li>
 
         <li>
-Each service endpoint MUST include <code>id</code>,
-<code>type</code>, and <code>serviceEndpoint</code> properties, and
-MAY include additional properties.
+Each service endpoint MAY include an <code>id</code> property.
+The array of service endpoints MUST NOT contain duplicate entries
```

Comment:
Again, should processors throw errors if they find duplicate entries? We should probably define a conformance class here for a DID Document processor and restate this as "A DID Document Processor MUST throw an error if there are duplicate entries with the same id." ... or something to that effect.

Reply:
Yes, I'd say throw an error in this case, as above. Regarding a "DID Document Processor" conformance class, do you want to do that here in this PR, or would that first have to be introduced/defined elsewhere?

Reply:
> Regarding a "DID Document Processor" conformance class, do you want to do that here in this PR, or would that first have to be introduced/defined elsewhere?

Ideally, we'd define a DID Document Processor conformance class first, merge that in, then fix this one and merge it in.

Alternatively, and I think this would be better, just raise an issue stating that we need to define a DID Document Processor conformance class, fix this PR so it throws an error, we'll merge this, and get to the DID Document Processor conformance class later. Better to get this PR merged than have it hang out there for too long.

Reply:
Just added text about producing errors in https://github.com/w3c/did-spec/pull/66/commits/bb57e32755d621a03a9d5442d8f90786589c25d5. Also raised issue https://github.com/w3c/did-spec/issues/80.